content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -454,6 +454,7 @@ pluginapi.Device{ID: "25102017", Health: pluginapi.Healthy, Topology:&pluginapi.
 
 Here are some examples of device plugin implementations:
 
+* The [Akri](https://github.com/project-akri/akri) lets you easily expose heterogeneous leaf devices (such as IP cameras and USB devices).
```

Comment:
I might write:
```suggestion
* [Akri](https://github.com/project-akri/akri), which lets you easily expose heterogeneous leaf devices (such as IP cameras and USB devices).
```

Reply:
Sorry for missed this comment. I have created a [new PR](https://github.com/kubernetes/website/pull/45993) to address it.